pkg/core/config/main.go

```diff
@@ -777,7 +778,10 @@ func (config *Config) Update(data interface{}) (err error) {
 		return err
 	}
 
-	tmpl, err := template.New("cfg").Funcs(updatecliRuntimeFuncMap(data)).Parse(string(content))
+	tmpl, err := template.New("cfg").
+		Funcs(sprig.FuncMap()).
```

Comment:
While we are here, I am wondering if you shouldn't also add the custom Helm function
https://github.com/updatecli/updatecli/blob/7e7aaa2e622dc26f9fdfe8b58e228879afcdfda7/pkg/core/config/template.go#L49

```suggestion
		Funcs(sprig.FuncMap()).
				Funcs(helmFuncMap()).      // add helm funcMap
```